bootstrap/terraform-fully-private/README.md

```diff
@@ -0,0 +1,196 @@
+- Creates EKS Cluster Control plane with one managed node group
+- Crossplane Add-on to EKS Cluster
+- Upbound AWS Provider for Crossplane
+- AWS Provider for Crossplane
+- Kubernetes Provider for Crossplane
+- Helm Provider for Crossplane
+
+## Crossplane Deployment Design
+
+```mermaid
+graph TD;
+    subgraph AWS Cloud
+    id1(VPC)-->Private-Subnet1;
+    id1(VPC)-->Private-Subnet2;
+    id1(VPC)-->Private-Subnet3;
+    Private-Subnet1-->EKS{{"EKS #9829;"}}
+    Private-Subnet2-->EKS{{"EKS #9829;"}}
+    Private-Subnet3-->EKS{{"EKS #9829;"}}
+    EKS==>ManagedNodeGroup;
+    ManagedNodeGroup-->|enable_crossplane=true|id2([Crossplane]);
+    subgraph Kubernetes Add-ons
+    id2([Crossplane])-.->|crossplane_aws_provider.enable=true|id3([AWS-Provider]);
+    id2([Crossplane])-.->|crossplane_upbound_aws_provider.enable=true|id4([Upbound-AWS-Provider]);
+    id2([Crossplane])-.->|crossplane_kubernetes_provider.enable=true|id5([Kubernetes-Provider]);
+    id2([Crossplane])-.->|crossplane_helm_provider.enable=true|id6([Helm-Provider]);
+    end
+    end
+```
+
+## How to Deploy
+### Prerequisites:
+Ensure that you have installed the following tools in your Mac or Windows Laptop before start working with this module and run Terraform Plan and Apply
+1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+1. [Kubectl](https://Kubernetes.io/docs/tasks/tools/)
+1. [Terraform >=v1.0.0](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+
+### Troubleshooting
+1. If `terraform apply` errors out after creating the cluster when trying to apply the helm charts, try running the command:
+```shell
+aws eks --region <enter-your-region> update-kubeconfig --name <cluster-name> --alias <cluster-name>
+```
+and executing terraform apply again.
+
+1. Make sure you have upgraded to the latest version of AWS CLI. Make sure your AWS credentials are properly configured as well.
+
+### Deployment Steps
+#### Step 0: Clone the repo using the command below
+
+```shell script
+git clone https://github.com/awslabs/crossplane-on-eks.git
+```
+
+> [!IMPORTANT]  
+> The examples in this repository make use of one of the Crossplane AWS providers. 
+For that reason `upbound_aws_provider.enable` is set to `true` and `aws_provider.enable` is set to `false`. If you use the examples for `aws_provider`, adjust the terraform [main.tf](https://github.com/awslabs/crossplane-on-eks/blob/main/bootstrap/terraform/main.tf) in order install only the necessary CRDs to the Kubernetes cluster.
+
+#### Step 1: ECR Settings
+
+**Retrieve Docker Credentials:**
+   - Replace `your-docker-username` and `your-docker-password` with your actual Docker credentials. 
+   - To retrieve the Docker password for Amazon ECR, use the following command, making sure to replace `your-region` with the AWS region where your ECR repository is located:
+     ```bash
+     aws ecr get-login-password --region your-region
+     ```
```

Comment:
This is not the ecr password, it is the docker-hub login. The one you would use to login here: https://login.docker.com/u/login/identifier.

Reply:
I've reverted this change.